provider/connections.go

```diff
@@ -81,7 +81,7 @@ var (
 		func() datasource.DataSource { return &StripeConnectionDataSource{} },
 	}
 
-	connectionImportables = map[string]resource.Resource{
+	connectionImportableResourcess = map[string]resource.Resource{
```

Comment:
This will need to be modified in `provider/gen/connections.go.tmpl` to persist after running `go generate`

Reply:
Ah, thanks; fixed.